src/concat_text/script.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
```

Comment:
`set- e` has some problems: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html
```
The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, part of any command executed in a && or || list except the command following the final && or ||, any command in a pipeline but the last, or if the command’s return status is being inverted with !
```
So if using `a && b`, and a fails, the script will not exit.

I think its better to use `set -euo pipefail` or similar